eldritch/test_encounters.py

```diff
@@ -503,6 +775,115 @@ def testWitch2Fail(self):
       self.resolve_until_done()
     self.assertEqual(len(self.char.possessions), 0)
 
+  def testWitch3Zero(self):
+    self.char.sanity = 4
+    self.state.event_stack.append(encounters.Witch3(self.char))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=4)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 1)
+
+  def testWitch3ZeroInsane(self):
+    self.char.sanity = 3
+    self.state.event_stack.append(encounters.Witch3(self.char))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=4)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 1)
+    self.assertEqual(self.char.place.name, "Asylum")
+
+  def testWitch3One(self):
+    self.char.lore_luck_slider = 3
+    self.state.event_stack.append(encounters.Witch3(self.char))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.delayed_until, self.state.turn_number + 2)
+
+  def testWitch3Two(self):
+    self.char.lore_luck_slider = 2
+    self.char.stamina = 3
+    self.state.event_stack.append(encounters.Witch3(self.char))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 2)
+
+  def testWitch3Three(self):
+    self.char.lore_luck_slider = 1
+    self.char.stamina = 2
+    self.state.event_stack.append(encounters.Witch3(self.char))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 5)
+
+  def testWitch4(self):
+    self.char.sanity = 3
+    self.state.event_stack.append(encounters.Witch4(self.char))
+    self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 2)
+
+  def testWitch4(self):
```

Comment:
You have accidentally named these two tests the same - I believe only one of them will run.

Reply:
But what are the odds that clobbering the first test will mask that the test was asking if the character lost stamina instead of sanity? 
<img width="569" alt="image" src="https://user-images.githubusercontent.com/441101/117915810-dc8f0400-b29a-11eb-9717-77f419aea3fd.png">